nautobot/core/templates/generic/object_edit.html

```diff
@@ -11,11 +11,11 @@
         <div class="row">
             <div class="col-md-6 col-md-offset-3">
                 <h3>
-                    {% if settings.DOCS_ROOT %}
+                    {% if obj %}
```

Comment:
If we can make `get_docs_url` return None in the case where the appropriate static file doesn't exist (see my other comment), then we could do something here like:

```django
{% if obj %}
    {% with get_docs_url(obj) as docs_url %}
        {% if docs_url %}
            <div class="pull-right">
                ...
            </div>
        {% endif %}
    {% endwith %}
{% endif %}
```

to ensure that we only show the help button if this model actually has valid docs.

Reply:
Ah, good idea! This would be nice.

Reply:
See comment above,  perhaps we can do something a little different than checking the url, or checking the url with Client Side Scripting. I think this change would be out of scope of this PR though.